### ISSUE TITLE
ISOtime as future date

### ISSUE THREAD

**Issue by egencia-cmao-github** [hashicorp/packer#7085]
been using packer to build my AWS AMIs but want to use an ami clean up that's based on timestamp.
I know i can use {{isotime \"2006.01.02.150405\"}} as Date in TAG, is it possible to set 90 days from now using this method? Thanks

**Comment by azr** [comment]
TLDR: This should be easily achieved by adding an `isotime_plus` function which takes a duration. In the packer/template/interpolate/funcs.go

___

Hey @egencia-cmao-github, no and there is no such feature yet. `isotime` will just give you a 'now' that you can also format.

Here's it's current code:
https://github.com/hashicorp/packer/blob/ecc20cf81627c3de39747f34724fb987705e139d/template/interpolate/funcs.go#L113-L125


You're the first person I can think of that asked for this feature so I will say the emblematic: 'pull requests are welcome'.

🙂 

But ! This should be pretty easy to achieve !

By adding an `isotime_plus` function which takes a duration. In the same file that I linked.

**Comment by azr** [comment]
Ah I didn't think of that one yes; I agree with you @rickard-von-essen !
